dbfit-java/build.gradle

```diff
@@ -89,16 +89,19 @@ subprojects {
         classpath += sourceSets.main.runtimeClasspath
     }
 
-    ext.dbdeploy = { dbDriver, url, userid, password, delimiterType = 'normal' ->
+    ext.dbdeploy = {
+        dbDriver, url, userid, password, delimiterType = 'normal', delimiter = ';',
+        dir = "$projectDir/src/integration-test/resources" ->
```

Comment:
Will dbdeploy pick only script filenames starting with some digits (as we already have some .sql scripts under resources).